src/runtime/vdom/test/event-listeners.spec.ts

```diff
@@ -21,7 +21,7 @@ describe('event listeners', () => {
 
     const vnode = h('div', { onClick: clicked }, h('a', null, 'Click my parent'));
 
-    patch(vnode0, vnode, document);
+    patch(vnode0, vnode);
```

Comment:
I'm not sure why these were all receiving a third `document` argument still. It looks like an older version of the `patch` function took this argument but it was later removed, see https://github.com/ionic-team/stencil/commit/15c00c0f947f4976f3f0a465c147bd55c8382269